samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java

```diff
@@ -37,24 +40,21 @@ public static void main(String[] args) throws IOException {
   // credentials to use.
   public static void authenticateImplicitWithAdc(String project) throws IOException {
 
-    String zone = "us-central1-a";
-    // This snippet demonstrates how to list instances.
+    // This snippet demonstrates how to list storage buckets.
```

Comment:
Please remove-- while strictly true, it's not really what the snippet is about (adc).

Reply:
done.